### PR TITLE
feat(DropdownButton): add horizontal kebab

### PR DESCRIPTION
**Pull request by sooinc**
### Overview

- Add horizontalKebab button type

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/go1LZq5mhdpWQgBzQJfxpI/Career-Journey?node-id=508%3A22476
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.86-alpha.db2cfa.0<br/>@codecademy/gamut-labs@30.5.1-alpha.db2cfa.0<br/>@codecademy/styleguide@55.4.1-alpha.db2cfa.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623b63256cac3d135319f065--gamut-preview.netlify.app>https://623b63256cac3d135319f065--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623b63256cac3d135319f065>Deploy Logs</a></p>

**Comment by sooinc** [comment]
See the horizontal menu used in portal-app here:  https://github.com/codecademy-engineering/portal-app/pull/1679****
(note functionality is not hooked up yet)

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623c859203a0217e252297d7--gamut-preview.netlify.app>https://623c859203a0217e252297d7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623c859203a0217e252297d7>Deploy Logs</a></p>

File: packages/gamut-labs/src/DropdownButton/index.tsx [hunk 1]
```diff
@@ -18,8 +18,12 @@ const DownArrow = styled(ArrowChevronDownFilledIcon, styledOptions)<{
   ${({ isOpen }) => isOpen && 'transform: rotate(-180deg)'};
 `;
 
+const HorizontalKebabIcon = styled(MiniKebabMenuIcon)`
+  transform: rotate(90deg);
+`;
```
Comment (dreamwasp) [inline packages/gamut-labs/src/DropdownButton/index.tsx:23]: i think this should work with  

```
const HorizontalKebabIcon = styled(MiniKebabMenuIcon)css({
  transform: 'rotate(90deg)'
})
```

if swapping over to that has weird side effects don't worry about it!

**Review by dreamwasp — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.86-alpha.7f84a2.0<br/>@codecademy/gamut-labs@30.5.1-alpha.7f84a2.0<br/>@codecademy/styleguide@55.4.1-alpha.7f84a2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623ca11e4fc2f100800875f6--gamut-preview.netlify.app>https://623ca11e4fc2f100800875f6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623ca11e4fc2f100800875f6>Deploy Logs</a></p>

**Pull request merged**